### DISCUSSION TITLE
I can make some vector art for extension covers

### DISCUSSION THREAD

**Discussion by Flappy25** [TurboWarp/extensions#511] [General]
I can make vector for some of the extensions that do not have covers

**Comment by LilyMakesThings** [comment]
https://github.com/TurboWarp/extensions/issues/90

**Reply by Flappy25** [reply]
ok, thanks I will close this

**Comment by Flappy25** [comment]
need to close

**Comment by Flappy25** [comment]
close

**Reply by GarboMuffin** [reply]
the discussion is closed. you do not need to comment on it. just let it be
